iati/home/tests/test_middleware.py

```diff
@@ -0,0 +1,55 @@
+"""A module of unit tests for redirects."""
+import pytest
+from django.conf import settings
+from factories import DocumentFactory
+from wagtail.core.models import Site, Page
+from about.models import AboutPage
+
+
+@pytest.mark.django_db
+class TestRedirectMiddleware():
+    """Tests for the custom redirect middleware."""
+
+    def create_document(self):
+        """Call the DocumentFactory to save a new document."""
+        return DocumentFactory.create()
+
+    def test_redirect_middleware_internal(self, client, rf):
+        """
+        Test behavior for internal redirects.
+
+        Create about page not dependent on default pages being added
+        via management command.
+        """
+        client_request = rf.get('/', follow=True)
+        site = Site.find_for_request(client_request)
+        site.hostname = client_request.get_host()
+        site.save()
+        home = Page.objects.get(id=3)
+        about = AboutPage(title='About us')
```

Comment:
@Ocre42 
Adding a new instance of the About page, so that this test is not reliant on `createdefaultpages` being run beforehand